phpstan.neon

```diff
@@ -20,3 +20,5 @@ parameters:
         console_application_loader: Tests/phpstan/console-application.php
     doctrine:
         objectManagerLoader: Tests/phpstan/object-manager.php
+    checkMissingIterableValueType: false
```

Comment:
would prefer to not disable these checks 🙂

Reply:
fixed